main.go

```diff
@@ -81,6 +83,11 @@ func doMain(args []string) {
 				Usage:  "Run the migrations",
 				Action: cmdMigrate,
 			},
+			{
+				Name:   "list-jobs",
+				Usage:  "list jobs",
+				Action: cmdListJobs,
+			},
```

Comment:
Nice idea the list-jobs, but I believe we need pagination (see comment below)

Reply:
sure [added](https://github.com/mendersoftware/workflows/pull/39/commits/a2882aa80c56c446976cc0d5028bcaa4ce6a36f6#diff-7ddfb3e035b42cd70649cc33393fe32cR181)

Reply:
TOP!